extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -14,23 +14,112 @@
  * limitations under the License.
  */
 
+.i-amphtml-story-interactive-slider-container {
+  --percentage: 0%;
+}
+
 .i-amphtml-story-interactive-slider-input-container {
+  display: flex !important;
   background: var(--i-amphtml-interactive-option-background-color) !important;
   border-radius: 2em !important;
   box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
 }
 
+.i-amphtml-story-interactive-slider-bubble {
+  position: absolute !important;
+  transform: translate(-50%, -50%) scale(0);
+  width: 2.8em;
+  height: 2.8em;
+  top: 50%;
+  left: var(--percentage);
+  text-align: center;
+  line-height: 2.6em;
+  z-index: 2 !important;
+  font-weight: 500 !important;
+  color: #fff !important;
+  transition: transform 0.3s ease-in-out !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble::before {
+  text-align: center;
+  line-height: 2.8em;
+}
+
+.i-amphtml-story-interactive-slider-bubble.show {
+  transform: translate(-50%, -170%) scale(1) !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble::after {
+  position: absolute !important;
+  content: '' !important;
+  box-sizing: border-box !important;
+  height: 2.8em !important;
+  width: 2.8em !important;
+  background: var(--interactive-accent-color) !important;
+  z-index: -1 !important;
+  left: 50% !important;
+  top: -0.1em !important;
```

Comment:
This looks hacky, do we need it for any known reasons?

Reply:
Yes it centers the post state bubble, brings it to the front and makes sure that it's background color matches the accent color (otherwise it is transparent)

Reply:
I mean just the `top: -0.1em` line, the rest is good 😁

Reply:
The top: -0.1em line centers the post state bubble, by shifting the bubble up slightly. It is a minor shift, but I think it makes the post state bubble more appealing and more on par with the figma file

Reply:
This might be a pixel shifting due to responsive values. It's noticeable since the elements are small. 
When resizing the window sometimes it looks a little heigh, a little low, or directly centered.  

I'm not sure that adding the `top: -0.1em` is solving this.

I wonder if putting `line-height: 2.9em;` on `.i-amphtml-story-interactive-slider-bubble` gives a better responsive experience.